core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.html

```diff
@@ -56,6 +56,28 @@ <h3>Generate dummy explorations</h3>
     </div>
   </mat-card>
 
+  <mat-card class="oppia-page-card oppia-long-text oppia-long-text-card">
+    <div class="container">
+      <h3>Generate dummy explorations</h3>
```

Comment:
We already have an `<h3>` element called "Generate dummy explorations" in the same component. Can we call it "Generate dummy translation opportunities" instead?

Reply:
Yes, I actually missed to change that. I'll update it.